src/main/java/elements/PageElementList.java

```diff
@@ -0,0 +1,157 @@
+package elements;
+
+import factory.DriverContainer;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.*;
+
+public class PageElementList implements List<PageElement> {
```

Comment:
Split in to interface and class realization

Reply:
fixed